sys/log/full/src/log_fcb.c

```diff
@@ -159,6 +159,11 @@ log_fcb_start_append(struct log *log, int len, struct fcb_entry *loc)
         }
 #endif
 
+        /* Check to see if the active region is in the scratch region */
```

Comment:
I think you're here if a flash area is about to get erased. Comment implies that this might not be the case.

Reply:
Updated comment.